Android/ChorusRFLaptimer/app/src/main/res/layout/race_setup.xml

```diff
@@ -375,6 +375,37 @@
 
         </LinearLayout>
 
+        <LinearLayout

```

Comment:
let's move it right after the "speak lap times" checkbox, since these are dependent ones